showthread.php

```diff
@@ -734,15 +734,18 @@
 	}
 
 	// Increment the thread view.
-	if($mybb->settings['delayedthreadviews'] == 1)
+	if(!($mybb->settings['threadviewcountexcludespiders'] == 1 && $session->is_spider == true || $mybb->settings['threadviewcountexcludeguests'] == 1 && $mybb->user['uid'] == 0 || $mybb->settings['threadviewcountexcludethreadauthor'] == 1 && $mybb->user['uid'] == $thread['uid']))
```

Comment:
```suggestion
	if(!(
		($mybb->settings['threadviewcountexcludespiders'] == 1 && $session->is_spider == true) ||
		($mybb->settings['threadviewcountexcludeguests'] == 1 && $mybb->user['uid'] == 0) ||
		($mybb->settings['threadviewcountexcludethreadauthor'] == 1 && $mybb->user['uid'] == $thread['uid'])
	))
```
Split + parentheses for clarity